frontend/src/app/app.component.css

```
@@ -0,0 +1,24 @@
+.app-container {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+}
+
+.main-content {
```

Würde es sinn machen denn content immer den ganzen platz zu geben damit der footer immer unten bleibt auch bei zu "kleinen" Seiten?
```suggestion
.main-content {
 height: 100svh;
 height: 100vh;
```